src/components/layouts/Layout.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from "@emotion/styled";
+import React from "react";
+import { Navigation } from ".";
```

Comment:
p3: 점이 무엇을 말하는 것인지 헷갈립니다. 혹시 같은 폴더내 컴포넌트내서 import하는 거라면  아래처럼 명시적으로 슬래쉬로 적어줘도 될것 같습니다.
```suggestion
import { Navigation } from "./";
```

Reply:
명시적으로 수정했습니다!